crates/topos-tce-broadcast/benches/double_echo.rs

```diff
@@ -9,7 +9,7 @@ pub fn criterion_benchmark(c: &mut Criterion) {
         .build()
         .unwrap();
 
-    let (_, store) = runtime.block_on(async { create_validator_store("benchmarks", vec![]).await });
+    let store = runtime.block_on(async { create_validator_store::partial_1(vec![]).await });
```

Comment:
What is this `partial_1`? Is it created at compile time?

Reply:
it is an automatic generated method for every function that derive `rstest::fixture`